.nvmrc

```diff
@@ -1 +1 @@
-14.18.2
+v14.21.3
```

Comment:
Thanks @rgarner

Would you mind setting this to allow any Node.js v14 LTS release?

It's something we've started doing on our other repos

```suggestion
lts/fermium
```

Reply:
Oh, that's much better, thanks. I had no idea.